documentation/book/con-kafka-listeners.adoc

```diff
@@ -12,7 +12,7 @@ The `listeners` property contains three sub-properties:
 * `tls`
 * `external`
 
-When none of these properties are defined, the listener will be disabled.
+When the property for given listener is not defined, the listener is disabled.
```

Comment:
It's a bit weird to talk about "the given listener" while simultaneously talking about that thing being absent.

```suggestion
Each listener will only be defined when the `listeners` object has the given property.
```